python/mlcroissant/mlcroissant/_src/core/rdf.py

```diff
@@ -26,8 +26,6 @@ def make_context(ctx=None, **kwargs):
         "conformsTo": "dct:conformsTo",
         "cr": "http://mlcommons.org/croissant/",
         "data": {"@id": "cr:data", "@type": "@json"},
-        "dataBiases": "cr:dataBiases",
```

Comment:
Should these still be defined in the context with the rai prefix?

Reply:
No, I will remove it.